### PR TITLE
Freshness pass for the app-only authentication article.

### PR DESCRIPTION
**Pull request by msewaweru**
Work items: [83548](https://dev.azure.com/msft-skilling/Content/_workitems/edit/83548) and [53899](https://dev.azure.com/msft-skilling/Content/_workitems/edit/53899)

### CODE REVIEW

**Comment by msewaweru** [comment]
@jasonjoh, I have done a freshness pass on this article and I'm requesting for your review on the same. Thanks

File: microsoftgraph/docs-conceptual/app-only.md [hunk 1]
```diff
@@ -18,16 +18,16 @@ The PowerShell SDK supports two types of authentication: [delegated access](/gra
...
-Before you can use app-only access with the SDK, you need the following.
+Before you can use app-only access with the Microsoft Graph PowerShell SDK, make sure you have the required prerequisites:
 
-- A certificate to use as a credential for the application. This certificate can be a self-signed certificate or a certificate from an authority. Refer to the [See also](#see-also) section for guidance on how to create a self-signed certificate.
-- [Register an application](/azure/active-directory/develop/app-objects-and-service-principals) in Azure AD, configure it with the permission scopes your scenario requires, and share the public key for your certificate.
-
-### Certificate
-
-You'll need an X.509 certificate installed in your user's trusted store on the machine where you'll run the script. You'll also need the certificate's public key exported in .cer, .pem, or .crt format. You'll need the value of the certificate subject or its thumbprint.
+1. Microsoft Graph PowerShell SDK is installed. Follow the [Install the Microsoft Graph PowerShell SDK](installation.md) guide to install the SDK.
```
Comment (jasonjoh) [inline microsoftgraph/docs-conceptual/app-only.md:25]: I'm not sure a numbered list is appropriate here. These aren't ordered steps. Just a suggestion, but I would leave this as bullets.

**Review by jasonjoh — APPROVED** [review]

File: microsoftgraph/docs-conceptual/app-only.md (hunk 1 above)
Comment (msewaweru) [reply microsoftgraph/docs-conceptual/app-only.md]: Thank you, @jasonjoh. I have adopted the suggestion. Appreciate it!

**Comment by msewaweru** [comment]
Hi @timayabi2020, please merge this.

**Review by timayabi2020 — APPROVED** [review]

**Pull request merged**